library/ctr_drbg.c

```diff
@@ -457,10 +456,11 @@ int mbedtls_ctr_drbg_random_with_add( void *p_rng,
 exit:
     mbedtls_zeroize( add_input, sizeof( add_input ) );
     mbedtls_zeroize( tmp, sizeof( tmp ) );
-    return( 0 );
+    return( ret );
 }
 
-int mbedtls_ctr_drbg_random( void *p_rng, unsigned char *output, size_t output_len )
+int mbedtls_ctr_drbg_random( void *p_rng, unsigned char *output,
+                             size_t output_len )
```

Comment:
I do not think this has anything to do with the change of the main PR, but its just style so I dont mind.

Reply:
Yes, it was a mistake in resolving the cherry-picking